qiskit/qasm3/__init__.py

```diff
@@ -130,6 +130,7 @@
 convert it into a :class:`.QuantumCircuit`:
 
 .. plot::
+    :alt: Output from the previous code.
```

Comment:
I think this is a case of:

```suggestion
    :alt: Circuit diagram output by the previous code.
```

Reply:
I spoke to the accessibility team, and "Output from the previous code." was their suggestion, since it should be illustrating what we explain in the text and code. But I agree that it's always good to be as helpful as possible.